modules/tensor_mechanics/doc/content/modules/tensor_mechanics/1d_elastic_waves.md

```diff
@@ -1,101 +1,127 @@
 
-Frequency domain analysis is often used to determine a frequency response function (FRF). An FRF describes the relationship between an input (frequency and amplitude of the input forcing source) and output (displacement response of a system). For simple systems, an analytic FRF can be derived. For more complex systems, the FRF is numerically obtained by determining the system response over a range of frequencies. The frequencies correspond to the peaks in the response on the FRF plot indicate natural frequencies of the system (eigen/fundumental frequencies). The mode shape (eigenvector) is given by the displacement profile at a natural frequency.
+Frequency domain analysis is often used to determine a frequency response function (FRF). An FRF describes the relationship between an input (frequency and amplitude of the input forcing source) and output (displacement response of a system). For simple systems, an analytic FRF can be derived. For more complex systems, the FRF is numerically obtained by determining the system response over a range of frequencies. Peaks in the response on the FRF plot indicate natural frequencies of the system (eigen/fundumental frequencies). The mode shape (eigenvector) is given by the displacement profile at a natural frequency.
 
 Other applications of frequency domain dynamics are: (1) computation of band structure (dispersion curves) of lattices/metamaterial, (2) inverse design for vibration control, e.g. design a system so that it has as minimum/maximum response at particular frequency, (3) material properties inversion/optimization given discrete responses.
 
-On the other hand, frequency domain analyses can be advantageous over time domain counterpart in several cases, for example, when the frequency spectrum of a signal consists of a few frequencies, or, when it is needed to have a better control of the numerical dispersion.
+Frequency domain analyses can be advantageous over its time domain counterpart in several cases, for example, when the frequency spectrum of a signal consists of a few frequencies, or, when it is needed to have a better control of the numerical dispersion.
 
 # Problem Description
 
-A uniaxial beam, of length L=1 see [cantilever] subjected to time-harmonic displacement on right and has absobing boundary condition on left.
-
-!media media/tensor_mechanics/Cantilever_beam.png style=width:60%; caption=2D cantilever problem with a prescribed displacement boundary condition on the right end. id=cantilever
-
-The PDE that mathematically describes this time-harmonic structural dynamics problem is as follows:
+The equations of motion for an isotropic elastic solid is given by the following partial differential equation:
```

Comment:
I kept it in terms of displacement because it is easier to show the fourier transform and u turning into U

Reply:
Ok, we just need to state that "..with constant coefficient/Young's Modulus" because this PDE will only be valid for constant E.

Reply:
By constant E, do you mean constant in time?  That should be fine with how I have it written since it's elastic.  Maybe I should say small strain linear elasticity or something to indicate this is only good for linear problems.

Reply:
E can vary in space.  That will be handled correctly

Reply:
> By constant E, do you mean constant in time? That should be fine with how I have it written since it's elastic. Maybe I should say small strain linear elasticity or something to indicate this is only good for linear problems.

I mean, if E is spatially varying, then div (E grad (u)) is the correct way to write it, right? we only can get E out from the div( ) if E is constant in space. That's what I meant. It is just to be mathematically correct.